html/webappapis/dynamic-markup-insertion/opening-the-input-stream/resources/http-refresh.py

```diff
@@ -0,0 +1,5 @@
+import time
```

Comment:
Could dedupe with https://github.com/web-platform-tests/wpt/blob/master/html/semantics/document-metadata/the-meta-element/pragma-directives/attr-meta-http-equiv-refresh/support/refresh.py

Reply:
I didn't do that, as the tests in that directory tests relative URL resolution as well, and moving refresh.py would greatly complicate that.